gems/sorbet-runtime/test/types/props/constructor.rb

```diff
@@ -38,9 +54,10 @@ class Outer < T::Struct
   end
 
   it 'raises on unknown props' do
-    assert_raises(ArgumentError) do
-      MyStruct.new(notathing: 4)
+    err = assert_raises(ArgumentError) do
+      MyStruct.new(name: "Alex", notathing: 4)
     end
+    assert_equal("Opus::Types::Test::Props::ConstructorTest::MyStruct: Unrecognized properties: notathing", err.message)
```

Comment:
This test was passing but for the wrong reason. It was failing due to a `Missing required prop` argument error since `name` was missing. I fixed the test and explicitly tested the error message to ensure this is testing the right thing.